Makefile

```diff
@@ -39,6 +39,10 @@ help: ## Display this help.
 
 ##@ Development
 
+.PHONY: manifests
+manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```

Comment:
I think we can remove this target entirely as there's overlap with the `generate` target. I was under the impression that the issue here was that I had consolidated the generate/manifest targets into a single target, but forgot to remove any target that specifies the `manifests` target as a prequisite?